### DISCUSSION TITLE
Is it possible to disable codemirror trailing spaces highlighting?

### DISCUSSION THREAD

**Discussion by belonesox** [sagemathinc/cocalc#6648] [Q&A] [answered]
Is it possible to disable codemirror trailingspaces highlighting (without hacking code)?

![image](https://user-images.githubusercontent.com/1609739/233479494-e5aa62b0-783b-4516-96cb-81dced0545c0.png)

**Comment by williamstein** [comment, marked as answer]
Try the option "Show trailing whitespace: show spaces at ends of lines" in account preferences?   https://cocalc.com/settings

**Reply by belonesox** [reply]
Thank you very much! I was digging through the project/account settings, but somehow missed it. Would it be possible to unset the option by default? I care about my students.

For any Python code cell, it is really frustrating because every new empty line (due to auto-indent and spaces) becomes "red" and annoying.

**Reply by williamstein** [reply]
Yes -- see https://github.com/sagemathinc/cocalc/issues/6649
